src/Gui/Splashscreen.cpp

```diff
@@ -68,7 +68,7 @@ class SplashObserver : public Base::ILogger
 
         // allow to customize text position and color
         const std::map<std::string,std::string>& cfg = App::GetApplication().Config();
-        std::map<std::string,std::string>::const_iterator al = cfg.find("SplashAlignment");
+        auto al = cfg.find("SplashAlignment");

```

Comment:
Lost `const`-ness.

Reply:
`cfg` is `const`. So the compiler will make a `const-iterator `from `cfg.find()` and const-ness has not been lost, no?

Reply:
Yep, I overlooked the const above.